Utilities/Scripts/SlicerWizard/ExtensionProject.py

```diff
@@ -33,6 +33,8 @@ class ExtensionProject(object):
 
   _moduleInsertPlaceholder = "# NEXT_MODULE"
 
+  _referencedVariables = re.compile("\$\{([\w\_]+)\}")
```

Comment:
This doesn't seem right; don't you mean to use a raw string here? (Then, the `_` should not be escaped, and the other escapes will be seen by `re` as intended.)

Also, CMake for better or worse allows a lot more than `[\w_]` in variable names... not sure if that's important here or not, however.